### PR TITLE
Fixed not closing Toolbar etc. bug

### LINKED ISSUE #5645: Toolbar Menu Toggle and Outside Click Handling Bug
When you click on a toolbar icon, the menu that pops up isn't closing properly with a second click on the same icon. It's supposed to toggle open and closed each time you click, but right now, it just stays open. Secondly, our outside click feature, which is meant to close the menu if you click anywhere else on the screen, isn't working when you click on the user button. Ideally, clicking anywhere outside the menu should close it, but currently, clicking on the user button doesn't have this effect.

**Steps to reproduce the behavior:**

- Open the application and navigate to a page where the toolbar is present.
- Click on any toolbar icon to open the corresponding menu.
- Click on the same icon again. Observe that the menu does not close as expected.
- Now, with the menu open, click on the user button. Notice that the menu remains open instead of closing.-

**Expected behavior**
The menu should toggle (open/close) when the corresponding toolbar icon is clicked.
Any click outside of the open menu, including clicks on the user button, should close the menu.

**Screen Recording **
[sidebar open closing bug.webm](https://github.com/plone/volto/assets/138920477/997b439c-0f2c-4d37-9dce-742239a07301)

### PR DESCRIPTION
**Pull request by singhcoder694**
Fixes #5645 using the condition in handleoutsideClick and this can easily be done for other by just adding some other conditions if required, a quick and easy fix.

### CODE REVIEW

File: packages/volto/src/components/manage/Toolbar/Toolbar.jsx [hunk 1]
```diff
@@ -314,6 +314,9 @@ class Toolbar extends Component {
   };
 
   handleClickOutside = (e) => {
+    if (e.target.tagName==='svg'){
+      return;
```
Comment (stevepiercy) [inline packages/volto/src/components/manage/Toolbar/Toolbar.jsx:318]: This does not pass ESLint check. You need to fix this. See [Code quality](https://6.docs.plone.org/volto/contributing/index.html#code-quality)

File: packages/volto/news/5645.bugfix [hunk 2]
```diff
@@ -0,0 +1 @@
+Fixes Toolbar, More, Add and Outside Click Handling function Bug @singhcoder694
```
Comment (stevepiercy) [inline packages/volto/news/5645.bugfix:1]: This is a vague and useless news item. See https://6.docs.plone.org/contributing/index.html#change-log-entry

The format should be:

Changed [a thing] from [something] to [another something]. @singhcoder694

**Review by stevepiercy — CHANGES_REQUESTED** [review]

File: packages/volto/news/5645.bugfix (hunk 2 above)
Comment (ravjot07) [reply packages/volto/news/5645.bugfix:1]: Hello @singhcoder694 , I have already raised a PR for this issue soon after filing it.

Comment (singhcoder694) [reply packages/volto/news/5645.bugfix:1]: Ohhkkk.... u may proceed🙂

**Comment by ichim-david** [comment]
@singhcoder694 code does not follow quality assurance and doesn't solve the problem. Closing in favor of the work that is tried by the author of the issue https://github.com/plone/volto/issues/5645

**Pull request closed without merging**